mandatory/1-syntax-errors.js

```diff
@@ -19,24 +19,22 @@ function getTotal(a, b) {
 
 There are some Tests in this file that will help you work out if your code is working.
 
-To run these tests type `node 1-syntax-errors.js` into your terminal
+To run the tests for just this one file, type `npm test -- --testPathPattern 1-syntax-errors` into your terminal
+(Reminder: You must have run `npm install` one time before this will work!)
 
 ===================================================
 */
 
-const util = require('util');
+const {test} = require("@jest/globals");
```

Comment:
```suggestion
const {expect, test} = require("@jest/globals");
```

Reply:
This isn't _required_, but I can see the value of adding it for clarity... When `jest` runs individual test closures, it shoves a bunch of stuff into global scope. But I think it probably is worth explicitly importing, even though we haven't taught the trainees anything about `require` yet and are treating this set-up as a completely opaque box :)